code/__HELPERS/unsorted.dm

```diff
@@ -1355,3 +1355,12 @@ proc/pick_closest_path(value, list/matches = get_fancy_list_of_atom_types())
 	while(length(str) < 5)
 		str = "0" + str
 	. = str
+
+/atom/proc/Shake(pixelshiftx = 15, pixelshifty = 15, duration = 250)
+	var/initialpixelx = pixel_x
+	var/initialpixely = pixel_y
+	var/shiftx = rand(-pixelshiftx,pixelshiftx)
+	var/shifty = rand(-pixelshifty,pixelshifty)
+	animate(src, pixel_x = pixel_x + shiftx, pixel_y = pixel_y + shifty, time = 0.2, loop = duration)
```

Comment:
ideally we'd be able to randomize `shiftx` and `shifty` every time the animate is ran, since as it is Shake() just takes two values and moves the sprite back and forth between them, meaning this isn't a true shake. nobody was able to figure this out in coderbus, but we had some attempts involving addtimer, sleep, two animate()s, and for() loops.